app/views/grammars/index.html.erb

```diff
@@ -20,6 +21,7 @@
         <td><%= grammar.label %></td>
         <td><%= grammar.description %></td>
         <td><%= grammar.position %></td>
+        <td><%= grammar.examples.length %></td>
```

Comment:
I know you might hate me, but in general it's better to use `.size` in rails instead of `.length`